plugins/deps/src/main/java/tools.forma/deps/core/ConfigurationType.kt

```diff
@@ -1,38 +1,44 @@
 package tools.forma.deps.core
 
-import tools.forma.target.FormaTarget
 import java.io.File
+import tools.forma.target.FormaTarget
+
+sealed interface ConfigurationType {
```

Comment:
we have opportunity to name it not like in gradle, because name `configuration` is too confusing(IMO). What about `DependencyType` or something that can be tied with dependencies?

Reply:
I agree about the confusion on these terms but I don't want to solve it now :)